js/app.js

```diff
@@ -751,15 +752,35 @@ $(document).ready(function () {
           return;
         },
         onBeforeShow: function () {
-          $(this).ColorPickerSetColor(this.value);
+          $(this).ColorPickerSetColor(this.value.replace('#', ''));
         },
       })
       .bind(
         'keyup',
         /** @this HTMLElement */
         function () {
-          $(this).ColorPickerSetColor(this.value);
-          $(this).css('background-color', `#${this.value}`);
+          const setColorCode = `#${this.value.replace(/#/g, '')}`;
+          $(this).ColorPickerSetColor(setColorCode.replace('#', ''));
+          $(this).css('background-color', setColorCode);
+        },
+      )
+      .blur(
+        /** @this HTMLElement */
+        function () {
+          let displayColorCode = `#${this.value.replace(/#/g, '')}`;
```

Comment:
Could you strip off any whitespace-like characters using something like `.replace(/\s/g, '')`? In case there are leading or trailing spaces (or even spaces in between characters if a user makes a typo). Sometimes they are leading spaces/tabs in hex codes when a user copies them from some website.